lib/api.js

```diff
@@ -1,3 +1,4 @@
+// NOTE: it seems we need it otherwise Cypress mocks (cy.route) will not work
```

Comment:
This is good to know, cy.route is definitely integral to a few tests

Reply:
E2E 20 was failing and this is the reason I reverted this case.

I haven't looked at E2E 39 because it's skipped right now. It's likely that this will break E2E 39 then ...